Automattic/simplenote-android
# Update Note Editor

### Fix
Update managing tags in the note editor with interactive chips.  See the screenshots and animation below for illustration.

![update_note_editor_tags](https://user-images.githubusercontent.com/3827611/64477896-f5f2bb00-d15d-11e9-8c69-62194c67cfd3.png)

<a href="https://user-images.githubusercontent.com/3827611/64477898-f8edab80-d15d-11e9-8c2e-9d55939eaebd.gif"><img src="https://user-images.githubusercontent.com/3827611/64477898-f8edab80-d15d-11e9-8c2e-9d55939eaebd.gif" width="320"></a>

### Test
1. Tap any note in list with tags.
2. Notice tag interface at bottom of ***Edit*** tab.
3. Notice close button is hidden in tag chips.
4. Tap tag chip.
5. Notice close button is shown in tag chip from Step 4.
6. Tap close button in tag chip from Step 4.
7. Notice tag chip is removed.
8. Tap ***Add tag...*** field.
9. Enter first letter of tag chip from Step 4.
10. Notice popup menu shows tag(s) matching entered letter.
11. Tap tag in popup menu.
12. Notice tap from Step 11 added as tag chip.

### Review
Only one developer and one designer are required to review these changes, but anyone can perform the review.

### Release
These changes do not require release notes.
